ts/features/itwallet/common/components/discoveryBanner/ItwDiscoveryBanner.tsx

```diff
@@ -72,6 +72,16 @@ export const ItwDiscoveryBanner = ({
       title: I18n.t("features.itWallet.discovery.banner.home.title"),
       action: I18n.t("features.itWallet.discovery.banner.home.action")
     },
+    // TODO: Once offline access is fully integrated, the "homeWithOffline" copy can be merged into "home" and this dynamic logic can be removed.
```

Comment:
I would link a Jira Task and add a TODO comment in the other places we'll need to change after the offline integration-I guess we won't need `onboardingWithOffline` anymore, right?